ts/features/wallet/component/CardLogoPreview.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+import { Image, ImageSourcePropType, StyleSheet } from "react-native";
+import { CardLayoutPreview } from "./CardLayoutPreview";
+
+type Props = {
+  left: React.ReactNode;
+  image: ImageSourcePropType;
+  onPress?: () => void;
+};
+
+const styles = StyleSheet.create({
+  cardLogo: {
+    height: 30,
+    width: 48
+  }
+});
+
+/**
+ * A preview card that show as right section an image of fixed dimensions.
```

Comment:
```suggestion
 * A preview card that shows as right section an image of fixed dimensions.
```

Reply:
Fixed in [4baa476](https://github.com/pagopa/io-app/pull/2878/commits/4baa4762f718d987ebea2f98a9dbe161067f06ad)